apps/vs-code-designer/src/package.json

```diff
@@ -847,7 +847,7 @@
           },
           "azureLogicAppsStandard.autoBinariesInstallation": {
             "type": "boolean",
-            "description": "Start using binaries installation",
+            "description": "Start using binaries installation Test",
```

Comment:
@ecfan Could you please review this string?
This is the text for the configuration setting that will tell the user if they select the auto validation/installation of the binary dependencies